spring-cloud-skipper-shell-commands/src/test/java/org/springframework/cloud/skipper/shell/command/support/YmlUtilsTests.java

```diff
@@ -39,45 +34,44 @@ public void testSimpleConversion() {
 		assertThat(yml).isEqualTo("hello: oi\nworld: mundo\n");
 	}
 
-	@Test
-	public void testBasicContent() throws IOException {
-		Resource resource = new ClassPathResource(
-				"/org/springframework/cloud/skipper/shell/command/support/log4j.properties");
-		String stringToConvert = StreamUtils.copyToString(resource.getInputStream(), Charset.defaultCharset());
-		String yml = YmlUtils.convertFromCsvToYaml(stringToConvert);
-		Resource convertedYmlResource = new ClassPathResource(
-				"/org/springframework/cloud/skipper/shell/command/support/log4j.yml");
-		assertThat(yml).isEqualTo(StreamUtils.copyToString(convertedYmlResource.getInputStream(),
-				Charset.defaultCharset()));
-	}
-
 	@Test
 	public void testPropertiesParsingWithPackageDeps() throws IOException {
 		String properties = "log.spec.deploymentProperties.spring.cloud.deployer.cloudfoundry.route=mlp3-helloworld.cfapps.io,"
 				+ "time.spec.deploymentProperties.spring.cloud.deployer.cloudfoundry.route=mlp1-helloworld.cfapps.io";
-		String propertiesYml = YmlUtils.getYamlConfigValues(null, properties);
+		String propertiesYml = YmlUtils.convertFromCsvToYaml(properties);
 		assertThat(propertiesYml).isEqualTo(
 				"log:\n"
 						+ "  spec:\n"
-						+ "    deploymentProperties: {spring.cloud.deployer.cloudfoundry.route: mlp3-helloworld.cfapps.io}\n"
+						+ "    deploymentProperties:\n"
```

Comment:
These still need to be 'flattened' out, even though one starts with a root of `log` and the other a root of `time`.

Reply:
Just to keep it in mind, if starting from a YML document, we will need to also flatten out the deployment and app properties.  I thought there was an issue for this but can't find it.

Reply:
Ah my mistake, think I accidentally changed that test to a proper yml.

Reply:
Right, I need to change keyspace to a regex so that It can have optional parts. Was my original idea but went with a simpler route.